### DISCUSSION TITLE
Bot not giving roles

### DISCUSSION THREAD

**Discussion by Justinwasnothere** [danthe1st/Mee6-bypasser#65] [Support]
![image](https://user-images.githubusercontent.com/97201878/210012366-0d0f960b-dc18-45f2-816f-5d31952ee218.png)

I know it uses image recognition, my rank card is clear enough.
![image](https://user-images.githubusercontent.com/97201878/210012400-6cb82309-85b9-411d-bf1a-4d776b41da03.png)

**Comment by danthe1st** [comment]
If you are running on UNIX-like, systems, you might need to install `tesseract-ocr`.
On Debian-like systems, that would work with `sudo apt install tesseract-ocr`.

If that doesn't fix it, please include the whole stack trace where the exception occurred the first time (with all `Caused by` sections) from the log.
